docs/content/en/docs/tasks/cluster/cluster-upgrades.md

```diff
@@ -51,20 +51,28 @@ Anywhere components mentioned above to their latest available versions. All upgr
 __Upgrading an existing GitOps enabled cluster created by an older release of the EKS-A CLI (v0.5.0 and below) requires a manual update to the git repository structure.__ 
 	
 Before running the `upgrade` command on an existing GitOps enabled cluster, you must:
-- move the eksa cluster config into a subdirectory under the management cluster's root level, following the [repo directory convention]({{< relref "./cluster-flux/#create-gitops-configuration-repo" >}})
-- update the `clusterConfigPath` field of the `GitOpsConfig` of the `eksa-cluster.yaml` to reflect the new path of the `eksa-system` directory. For example:
-
-```yaml
-apiVersion: anywhere.eks.amazonaws.com/v1alpha1
-kind: GitOpsConfig
-metadata:
-  name: management-cluster
-spec:
-  flux:
-    github:
-      clusterConfigPath: clusters/management-cluster/management-cluster # old: clusters/management-cluster
-      ...
-```
+1. temporarily disable the EKS Anywhere webhook by 1) saving the eksa validating webhook locally 2) delete it in cluster
```

Comment:
sub-bullets?

Reply:
will it be better to reflect the order using 1.2.3.4?

Reply:
```suggestion
1. temporarily disable the EKS-A validating webhook 
    - save the eksa validating webhook configuration to a local file 
    - delete the validating webhook from the cluster
```

Reply:
ah!